NAMESPACE

```diff
@@ -65,6 +65,7 @@ export(getParameterDisplayPaths)
 export(getQuantity)
 export(getSimulationTree)
 export(getStandardMoleculeParameters)
+export(getUnitsForDimension)

```

Comment:
@abdullahhamadeh This will be soon available in ospsuite-r Returns all units for a given dimension name